state/protocol/badger/validity.go

```diff
@@ -197,6 +199,57 @@ func isValidEpochCommit(commit *flow.EpochCommit, setup *flow.EpochSetup) error
 	return nil
 }
 
+func isValidVersionBeacon(vb *flow.VersionBeacon, header *flow.Header) error {
+	if len(vb.RequiredVersions) == 0 {
+		return fmt.Errorf("required versions empty")
+	}
+
+	// first entry in a table must be a current version, so the height must be below the current block
+	if vb.RequiredVersions[0].Height > header.Height {
+		return fmt.Errorf("lowest required version height %d below current block's height %d", vb.RequiredVersions[0].Height, header.Height)
```

Comment:
```suggestion
		return fmt.Errorf("lowest required version height %d above current block's height %d", vb.RequiredVersions[0].Height, header.Height)
```
Error message doesn't match logic or comment.

I don't quite understand this condition though. When we require a new version, we would use a height a few thousand blocks in the future, to make sure all nodes have time to prepare. So, the first `VersionBeacon` event we process would be guaranteed to have some `Height > header.Height`, and therefore would always fail this check... 🤔

Reply:
First entry so first version, if present of course, should always be the current one - so it should be currently active and therefore the height must be at or below the current block (block at which we mark it as active)
It is to make sure we are on the proper version that interprets event as intended. 
Indeed, the error message was wrong, thanks for pointing that out.

Reply:
Resolving because we removed height-based validation